src/main.rs

```diff
@@ -486,6 +492,31 @@ fn run() -> Result<()> {
                 )
             })?;
 
+            if let Some(_) = app_matches.values_of("list languages") {
+                let languages = assets.syntax_set.syntaxes();
+
+                let longest = match languages.iter()
+                    .map(|s| s.name.len())
+                    .max() {
```

Comment:
you can use `languages.iter().....max().unwrap_or(32)` here instead of the `match` statement.